### PR TITLE
Add new testnet seednode

### PR DESCRIPTION
**Pull request by dev0tion**

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -169,7 +169,8 @@ class CTestNetParams : public CMainParams {
 
         vFixedSeeds.clear();
         vSeeds.clear();
-        vSeeds.push_back(CDNSSeedData("stratisplatform.com", "testnode1.stratisplatform.com"));
+        vSeeds.push_back(CDNSSeedData("testseednode1", "testnode1.stratisplatform.com"));
+        vSeeds.push_back(CDNSSeedData("testseednode2", "testnode2.stratis.cloud"));
```
Comment (bokobza) [inline src/chainparams.cpp:173]: you can even add testnode3 now I guess.

**Comment by dev0tion** [comment]
@dangershony

**Review by bokobza — COMMENTED** [review]
No testnode3?

**Comment by dev0tion** [comment]
@dangershony I see this code wasn't pushed in yet - it's still valid as we still have both seeders up and running.

**Review by dangershony — APPROVED** [review]

**Pull request merged**